index.html

```diff
@@ -1122,11 +1129,270 @@ <h5>
+          of user harm, such as an online store requesting users to sign up
+          with their driver's license instead of generic email & password or
+          federated credentials. This can lead to exclusion of users without
+          the ability or willingness to share such a credential with the site,
+          a deterioration of the prompt experience on the web, and an increase
+          in the risk of accidental data leakage.
+          </li>
+          <li>Requests for an excessive amount of information for valid
+          purposes against the principle of data minimization. A common example
+          is collection of a user's entire national identity document for age
+          verification instead of relying on selective disclosure.
+          </li>
+        </ul>
+        <p>
+          One challenge here is determining what constitutes "valid" purposes
+          and which requests are therefore "unnecessary", and requires
+          participation from all parties involved in the credential exchange.
+        </p>
+        <ul>
+          <li>Ideally, verifiers would self-regulate their requests for
+          credentials. However, from a user agent's perspective, verifiers are
+          potential attackers, and might not consider the user's best interest
+          in their designs.
+          </li>
+          <li>User agents are responsible for protecting their users against
+          dangerous content and permission requests on the Web and could
+          intervene on their behalf, proactively rejecting requests.
+          </li>
+          <li>Issuers and lawmakers might decide to restrict use of
+          (particularly government-issued) credentials to specific verifiers
+          with purpose attestations. Wallets might be expected to enforce these
+          restrictions.
+          </li>
+          <li>The ultimate decision of whether or not to share their personal
+          information lies with the user.
+          </li>
+        </ul>
+        <p>
+          For a more detailed exploration of how to determine and address
+          unnecessary usage, it makes sense to consider goverment-issued
+          credentials and other credentials separately, as they potentially
+          differ in the sensitivity of their data and the potential harms from
+          misuse as well as legal & regulatory considerations.
+        </p>
+        <p>
+          A key component of risk mitigation and ensuring user control that
+          applies to both types of credentials is the user agent's ability to
+          inspect the credential request metadata and make decisions or UI
+          presentation based on it. The DC API ensures this user agent access
+          to the exchange through requirements placed on protocols to transmit
+          requests unencrypted and to include relevant information.
+        </p>
+        <h4>
+          Government-issued credentials
```

Comment:
We don't need to address it immediately, but I suspect that the distinction here will not be entirely whether something is government-issued or not. There are other types of high-assurance credentials that are unique, cross-context, are issued by other authorities and have potential offline as well as online implications.

Related to #117 and #85.

Reply:
Thanks Kyle, I filed #295 for tracking your comment. I agree that this is a useful way to look at it but it might overload the PR if I try to incorporate it right now.